pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -116,6 +116,13 @@ type kubeGenericRuntimeManager struct {
 
 	// A shim to legacy functions for backward compatibility.
 	legacyLogProvider LegacyLogProvider
+
+	// cache of runtime configuration eg, user-namespace configuration
+	runtimeConfig *kubecontainer.RuntimeConfigInfo
+	// GID on host usernamespace mapped to GID 0 in container user-namespace
+	firstRemappedGIDOnHost int
```

Comment:
where is this configuration coming from ? kubelet configuration ?

Reply:
from runtime through CRI